pages/01.HDM Documentation/04.system-requirements/default.md

```diff
@@ -55,30 +43,38 @@ Related document: For hardware and system requirements, refer to the [VMware Har
 *   The PrimaryIO Appliance should be deployed on the same vCenter and datacenter it will be managing. It is recommended to deploy on a cluster that will not be migrated.
 *   HDM components should be deployed on a shared datastore.
 
+
 #### Cloud Requirements
 
+##### Cloud Environment
 *   Supported cloud endpoints:
-    *   VMware Cloud on AWS
-    *   IBM Cloud for VMware Solution
-    *   VMware Cloud Foundation SDDC
-    *   VMware Cloud Director
-*   HDM components should be deployed on Shared Datastore
-*   Network connectivity between on-premises and the cloud
-    *   Network Link >= 1 Gbps
-    *   Network Round Trip Time Latencies: &lt; 30 ms
-*   Network requirement:
-    *   Cloud HDM components should have access to the WAN network.
-    *   Cloud VCenter and ESXi should be accessible on-premises at port 443.
-    *   E1000E and VMXNet3 network adapter should be available in  the cloud vCenter.
-    *   Network on which HDM components are deployed should have access to the cloud vCenter at port 443 and the cloud ESXi at port 443 and 902.
-    *   Firewall rules for HDM cloud access:
-        *   Port 22 - inbound and outbound
-        *   Port 2379 - outbound 
-        *   Port 6000-6010 - inbound and outbound
-        *   Port 7000-7010 - inbound and outbound
-        *   Port 8000-8010 - inbound and outbound
-        *   Post 32820 - inbound and outbound
+   *   VMware Cloud on AWS
+   *   IBM Cloud for VMware Solution
+   *   VMware Cloud Foundation SDDC
+   *   VMware Cloud Director
+*   Shared Datastore for HDM components.
 
+##### Network Environment
+*   Cloud HDM components should have access to the WAN network.
+*   Cloud VCenter and ESXi should be accessible on-premises at port 443.
+*   E1000E and VMXNet3 network adapter should be available in  the cloud vCenter.
+*   Network on which HDM components are deployed should have access to the cloud vCenter at port 443 and the cloud ESXi at port 443 and 902.
+*   Firewall rules for HDM cloud access:
+    *   Port 22 - inbound and outbound
+    *   Port 2379 - outbound 
+    *   Port 6000-6010 - inbound and outbound
+    *   Port 7000-7010 - inbound and outbound
+    *   Port 8000-8010 - inbound and outbound
+    *   Post 32820 - inbound and outbound
+ 
+##### Network connectivity between on-premises and the cloud
+*   Network Link >= 1 Gbps
+*   Network Round Trip Time Latencies: &lt; 30 ms
+ 
+##### Hardware Requirements
+*   For HDM component requirements, check the deployment planning section as per your cloud end-point.
```

Comment:
Should there be link to point to the section ?

Reply:
Okay. we can put. Their will be 3 links as per cloud end-point.